assets/collage.css

```diff
@@ -98,12 +99,6 @@
     grid-column: 1 / span 3;
     max-width: 100%;
   }
-
-  .collage-card:not(.collage-card--left):not(.collage-card--right) > * {
-    position: absolute;
-    width: 100%;
-    height: 100%;
-  }
 }
```

Comment:
To add to this, the way the collage section was designed made it so that block being focused (left or right based on the option of the section settings), would not crop the image in most cases (unless it's too wide and not high enough). 
Right now when the focus is a product card or collection card, it will crop the image when it used to try and adjust the section size based on the dimension of the product/collection image. 

![](https://screenshot.click/21-11-ebzqv-5db1z.png)